Example mod/Example mod.csproj

```diff
@@ -32,7 +32,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Assembly-CSharp">
-      <HintPath>..\Dependencies\Assembly-CSharp.dll</HintPath>
+      <HintPath>..\..\DependenciesBZ\Assembly-CSharp.dll</HintPath>
```

Comment:
**TODO** 
Introduce two new [custom build configurations](https://docs.microsoft.com/en-us/visualstudio/ide/how-to-create-and-edit-configurations?view=vs-2019): `SUBNAUTICA`|`BELOWZERO`. 

Use these to [conditionally select](https://stackoverflow.com/questions/1786917/is-there-a-way-to-specify-assembly-references-based-on-build-configuration-in-vi) the correct Assembly-CSharp DLL for the game.